src/command_modules/azure-cli-network/azure/cli/command_modules/network/_params.py

```diff
@@ -417,15 +418,22 @@ def completer(prefix, action, parsed_args, **kwargs): # pylint: disable=unused-a
 register_cli_argument('network vpn-gateway', 'virtual_network_gateway_name', options_list=('--name', '-n'), completer=get_resource_name_completion_list('Microsoft.Network/virtualNetworkGateways'), id_part='name')
 register_cli_argument('network vpn-gateway', 'cert_name', help='Root certificate name', options_list=('--name', '-n'))
 register_cli_argument('network vpn-gateway', 'gateway_name', help='Virtual network gateway name')
+register_cli_argument('network vpn-gateway', 'gateway_type', help='The gateway type.', **enum_choice_list(gatewayType))
+register_cli_argument('network vpn-gateway', 'sku', help='VPN gateway SKU.', **enum_choice_list(sku))
+register_cli_argument('network vpn-gateway', 'vpn_type', help='VPN gateway type.', **enum_choice_list(vpnType))
```

Comment:
I renamed it for the custom command, but left the fat client template alone for fear that it would require me to re-record the 1-2 hour long test.